NEWS.rst

```diff
@@ -30,7 +30,9 @@ Other Breaking Changes
 * `hy-repr` uses registered functions instead of methods
 * `HyKeyword` no longer inherits from the string type and has been
   made into its own object type.
+* `HySymbol` no longer inherits from `HyString`.
 * `(except)` is no longer allowed. Use `(except [])` instead.
+* `(import [foo])` is no longer allowed. Use `(import foo)` instead.
```

Comment:
If we're going to do this, maybe it makes sense to go ahead and remove the second level nesting of arrays: `(import [foo bar baz])` now means `from foo import bar, baz`, instead of the existing `(import [foo [bar baz]])`

Reply:
Not a bad idea, but it can be done in a separate PR.

Reply:
@vodik The `import` syntax has bothered me for a while. See #851. `require` too for that matter.